src/pyscaffold/utils.py

```diff
@@ -295,3 +298,38 @@ def dasherize(word):
         input word with underscores replaced by dashes
     """
     return word.replace('_', '-')
+
+
+def get_id(function):
```

Comment:
Well, since we are on python3-only now, we could just use `__qualname__`. It would be longer, but do the job nicely.

Reply:
Good point, I removed `get_id`.

Reply:
Hm.. just realised that `__qualname__` does not include the module name. So I kept `get_id`.

Reply:
Sorry for the misleading comment :(
I was under the impression that it would include.